less/pageNav.less

```diff
@@ -28,6 +28,10 @@ html {
 .pagenav {
   &__btn {
     margin: @item-margin;
+
+    &[aria-current="page"] {
+      // TODO: put button styles for current page here
+    }
```

Comment:
This is unnecessary.

Reply:
Shouldn't this plugin provide some basic styles? Or is that up to the theme to come up with?

Reply:
Absolutely, on both counts. If you want to provide some basic styling here then put it in, or you can leave it to the theme. Having a comment leaves the situation in limbo.

Reply:
> What are the default styles for the depressed button state? 

A sad face icon?

I don't think we have any at the moment. It might just be the highlight/inverted colour thinking about it, resources tabs buttons may be the only example of "currently selected" on a button.

@kirsty-hames could you interject with a suggestion please?

Reply:
Looking at Vanilla colour variables, there are two instances of 'selected' - [@drawer-item-selected](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/bb314efec4b6736a435efd2ecec50a8fd8b97d2c/less/_defaults/_colors.less#L162) and [@item-color-selected](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/bb314efec4b6736a435efd2ecec50a8fd8b97d2c/less/_defaults/_colors.less#L42) both of which just darken their default colour variable by 20% so I think we can just do the same here for consistency?

e.g. `background-color: darken(@btn-item-color, 20%);`